homeassistant/components/sensor/repetier.py

```diff
@@ -0,0 +1,201 @@
+"""
+Sensor for retrieving Repetier-Server device status.
+
+Creates one sensor for each device attached to Repetier-Server
+Created by Morten Trab (morten@trab.dk) - 2019
+"""
+from datetime import timedelta
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (
+    CONF_API_KEY,
+    CONF_PORT,
+    CONF_URL,
+    STATE_IDLE,
+    STATE_OFF,
+    STATE_ON,
+    STATE_UNKNOWN)
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+
+REQUIREMENTS = ['pyrepetier==1.4.1']
+
+__version__ = '0.5.1'
```

Comment:
What is this version representing? Home Assistant components don't have standalone versions, so this should likely be removed.